src/main/resources/filter/web/visibility.filter

```diff
@@ -0,0 +1,188 @@
+# Visibility filter file for recheck.
+
+# Ignore visibility CSS attributes:
+
+attribute=align-content
+attribute=align-items
+attribute=align-self
+attribute=all
```

Comment:
I think `all` is a retest internal attribute. I don't think that is an existing CSS attribute?